.github/workflows/static-analysis.yml

```diff
@@ -9,9 +9,11 @@
 # the `language` matrix defined below to confirm you have the correct set of
 # supported CodeQL languages.
 #
```

Comment:
[optional] if we repurpose this file, IMO we can remove the comment boilerplate from the top of the file.

Alternatively: we keep the boilerplate and move govulncheck to a new file.

Reply:
sure, good idea :+1: 

Originally, I opened this PR to test vulncheck and see what the results it will give and if it's worth it to add it next to codeQL

Reply:
I expect go vulncheck will become a best practice for repos so I think we should add it.